notification_service/src/main/java/com/eai/notification_service/dto/EmailSenderRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.eai.notification_service.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EmailSenderRequest {
+    private String code_otp_email;
```

Comment:
This is a general EmailSenderRequest, which contains mail components (senderEmail, receiverEmail, subject, and text/body template).